setup.py

```diff
@@ -9,20 +9,20 @@ def read(fname):
 
 
 extras = {
-    "standard": ["pandas", "rtree", "pyyaml", "snakemake"],
     "dev": ["pytest", "black==20.8b1", "coverage", "flake8", "geopandas"],
 }
 
-extras["full"] = list(set(extras["standard"] + extras["dev"]))
-
-
 setup(
     name="oemof-B3",
     version="0.0.0",
     description="",
     long_description=read("README.md"),
     packages=["oemof_b3"],
     install_requires=[
+        "pandas",
+        "rtree",
+        "pyyaml",
+        "snakemake<6.0.0",
```

Comment:
This reverts the changes made in PR https://github.com/rl-institut/oemof-B3/pull/118, hence the checks for python 3.8 fail, as they did before the (dirty) fix for #114.